ipfs-cluster-ctl/main.go

```diff
@@ -249,12 +249,17 @@ peers should pin this content.
 							Value: 0,
 							Usage: "Sets a custom replication factor for this pin",
 						},
+						cli.StringFlag{
+							Name:  "name, n",
+							Value: "",
+							Usage: "Sets a name for this pin",
+						},
 					},
 					Action: func(c *cli.Context) error {
 						cidStr := c.Args().First()
 						_, err := cid.Decode(cidStr)
 						checkErr("parsing cid", err)
-						query := fmt.Sprintf("?replication_factor=%d", c.Int("replication"))
+						query := fmt.Sprintf("?replication_factor=%d&name=%s", c.Int("replication"), c.String("name"))
```

Comment:
Need to do url-escaping stuff with name, otherwise some names are gonna go wrong...